bot/cogs/cyber.py

```diff
@@ -34,21 +35,21 @@ def contains_words(self, text: str, all_words: Iterable[str], *any_words: Iterab
         """
 
         # Gather HQ data from CyberStart Game.
-        with open("headquarters.txt", "r") as f:
-            game_docs = [level.split(":::\n") for level in f.read().split(";;;;;;\n")]
+        with open("headquarters.json", "r") as f:
```

Comment:
calls to the `open` function don't require `'r'` to be passed as an argument - that's the default.